demo/radio-button-basic-demos.html

```diff
@@ -5,22 +5,93 @@
         display: block;
       }
     </style>
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet id="basic-usage">
+      <template preserve-content>
+        <vaadin-radio-group label="Label" theme="vertical">
+          <vaadin-radio-button checked>Option one</vaadin-radio-button>
+          <vaadin-radio-button>Option two</vaadin-radio-button>
+          <vaadin-radio-button>Option three</vaadin-radio-button>
+        </vaadin-radio-group>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Disabled and disabled item</h3>
+    <vaadin-demo-snippet id="disabled-and-disabled-item">
+      <template preserve-content>
+        <vaadin-radio-group label="Disabled" disabled theme="vertical"
+        >
+        <vaadin-radio-button checked>Option one</vaadin-radio-button>
+        <vaadin-radio-button>Option two</vaadin-radio-button>
+        <vaadin-radio-button>Option three</vaadin-radio-button>
+      </vaadin-radio-group>
 
+      <span style="margin-right: 200px;"></span>
```

Comment:
You can remove this also when removing `theme="vertical"`

Reply:
It's related to my previous comments. I already discussed it with Jouni and it is fine.